include/swift/Basic/Sanitizers.h

```diff
@@ -21,6 +21,8 @@ enum class SanitizerKind : unsigned {
   Thread,
 };
 
+enum class SanitizerCoverageKind : unsigned { None = 0, Function, BB, Edge };
```

Comment:
Perhaps a solution to the formatting quandary is to add comments describing the coverage kinds?